examples/todo/js/subscriptions/AddTodoSubscription.js

```diff
@@ -12,11 +12,8 @@ export default class AddTodoSubscription extends RelaySubscriptions.Subscription
   getSubscription() {
     return Relay.QL`subscription {
       addTodoSubscription {
-        clientMutationId
         todoEdge {
-          __typename
           node {
-            __typename
```

Comment:
Hmm maybe it is Relay that adds it then..? Relay couldn't get the type of the node so it didn't know how to cache it or something